scripts/transform_json.js

```diff
@@ -18,11 +19,25 @@ const transformFrameworks = async function () {
   await fs.writeFile(FRAMEWORKS_BUILD, fileContents)
 }
 
+const updateLogoUrls = function (contents) {
+  const updatedContents = contents
+  const originalLogo = contents.logo
+  if (originalLogo) {
+    for (const [theme, urlPath] of Object.entries(originalLogo)) {
+      updatedContents.logo[theme] = (process.env.DEPLOY_PRIME_URL || 'https://framework-info.netlify.app') + urlPath
```

Comment:
[dust] Prefer destructuring.

```suggestion
  const{ logo } = contents
  if (logo) {
    for (const [theme, urlPath] of Object.entries(logo)) {
      updatedContents.logo[theme] = (process.env.DEPLOY_PRIME_URL || 'https://framework-info.netlify.app') + urlPath
```

Reply:
[dust] Similar comment to @nickytonline's about destructuring - I prefer template strings rather than string appends. Perhaps we codify stuff like this in our ESLint config if we don't have rules for this already.